app/services/item_service.rb

```diff
@@ -0,0 +1,19 @@
+class ItemService 
+
+  def self.connection 
+    Faraday.new(url: "http://localhost:3000/api/v1/")
+  end
```

Comment:
This method is exactly the same in `MerchantService`. Could it be abstracted into something like `RailsEngineService`, so that we could DRY things up and have the independent existing services inherit the behavior?